extensions/Worker.Extensions.Kafka/release_notes.md

```diff
@@ -0,0 +1,10 @@
+## Release notes
+<!-- Please add your release notes in the following format:
+- My change description (#PR/#issue)
+-->
+
+### 3.5.0
+- Added the improvement for the producer logging to app insights & filesystem.
+- Added the support for the retry feature
+- Added the support for keys for triggers & output bindings
+- Added the support for auto.offset.reset config
```

Comment:
Thanks! Feel free to add a link to the issue/PR (see line 3 for format).

Reply:
Done